### ISSUE TITLE
Accessibility: Cannot turn off audio cues on a language level

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#185565]
CC @meganrogge

Tested on Windows. I cannot turn off audio cuse on a language level.

I don't want to get the audioCues.lineHasError played in markdown files, but it is still played.

`settings.json`:

``` json
{
  "audioCues.lineHasError": "off",
  "[markdown]": {
    "audioCues.lineHasError": "off"
  }
}
```

## System Info

Version: 1.80.0-insider (system setup)
Commit: 73eb619afb87cab4aa83a569259c891b6983c11e
Date: 2023-06-19T05:35:37.388Z
Electron: 22.3.11
ElectronBuildId: 21658788
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Comment by meganrogge** [comment]
Has this ever worked @hediet ?

I am afk but don't think so, in which case this would be a feature request.

**Comment by hediet** [comment]
Probably this never worked.
